jwst/regtest/test_niriss_image.py

```diff
@@ -56,7 +57,8 @@ def run_detector1_multiprocess_rate(rtdata_module):
             "--steps.charge_migration.skip=False",
             "--steps.charge_migration.save_results=True",
             "--steps.jump.save_results=True",
-            "--steps.ramp_fit.maximum_cores=2", # Multiprocessing
+            "--steps.ramp_fit.algorithm=OLS_C",  # currently default
```

Comment:
```suggestion
            "--steps.ramp_fit.algorithm=OLS_C",
```
This is the type of comment that I'm worried will quietly become out-of-date and no longer true.  Also it's not clear to me the benefit to the developer to have that information in this location